app/src/main/java/com/picone/go4lunch/presentation/ui/fragment/MapsFragment.java

```diff
@@ -127,4 +160,22 @@ private void updateLocationUI() {
             Log.e("Exception: %s", Objects.requireNonNull(e.getMessage()));
         }
     }
+
+
```

Comment:
What if you do BitmapFactory.decodeResource(resources, R.drawable.id) ?

Reply:
error :
D/skia: --- Failed to create image decoder with message 'unimplemented'
W/System.err: io.reactivex.exceptions.OnErrorNotImplementedException: Failed to decode image. The provided image must be a Bitmap.